frontend/src/api/ClientsRepository.js

```diff
@@ -10,6 +10,11 @@ export default {
     getRoles(clientId){
         return kcRequest().then(axiosInstance => axiosInstance.get(`${resource}/${clientId}/roles`));
```

Comment:
Does this API exist in the UMS now? If so, we should use it instead of the Keycloak API. Then ClientsRepository no longer depends on the KC API.